geode-core/src/main/java/org/apache/geode/distributed/internal/membership/InternalDistributedMember.java

```diff
@@ -272,10 +271,8 @@ public DurableClientAttributes getDurableClientAttributes() {
     return durableClientAttributes;
   }
 
-  /**
-   * Returns an unmodifiable Set of this member's Roles.
-   */
   @Override
+  @Deprecated
```

Comment:
A comment will be helpful mentioning what alternative to use instead of the deprecated

Reply:
The deprecation is documented in the interface this method derives from. Unfortunately the author did not provide an alternative. Best I can assume from the documentation is that the concept is unused and there is no replacement.